docs/development/extensions-core/kafka-ingestion.md

```diff
@@ -2,6 +2,7 @@
 id: kafka-ingestion
 title: "Apache Kafka ingestion"
 sidebar_label: "Apache Kafka"
```

Comment:
A small thing, but I'd suggest making the sidebar title to be the same as the page title here, just to make the distinction with the next two pages in the tree, which also start with "Apache Kafka" 

```suggestion
sidebar_label: "Apache Kafka ingestion"
```